docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1072,212 +1072,621 @@ thor:
-          good amount more memory or another for more CPU. You can use
-          placements to ensure that pods with specific requirements are placed
-          on appropriate nodes.</para>
+            <tbody>
+              <row>
+                <entry>Type: &lt;component&gt;</entry>
+
+                <entry>Covers all pods/jobs under this type of component. This
+                is commonly used for HPCC Systems components. For example, the
+                <emphasis>type:thor</emphasis> which will apply to any of the
+                Thor type components; thoragent, thormanager, thoragent and
+                thorworker, etc.</entry>
+              </row>
+
+              <row>
+                <entry>Target: &lt;name&gt;</entry>
+
+                <entry>The "name" field of each component, typical usage for
+                HPCC Systems components referrs to the cluster name. For
+                example <emphasis>Roxie: -name: roxie</emphasis> which will be
+                the "Roxie" target (cluster). You can also define multiple
+                targets with each having a unique name such as "roxie",
+                "roxie2", "roxie-web" etc.</entry>
+              </row>
+
+              <row>
+                <entry>Pod: &lt;name&gt;</entry>
+
+                <entry>This is the "Deployment" metadata name from the name of
+                the array item of a type. For example, "eclwatch-", "mydali-",
+                "thor-thoragent" This can be a regular expression since
+                Kubernetes will use the metadata name as a prefix and
+                dynamically generate the pod name such as,
+                eclwatch-7f4dd4dd44cb-c0w3x.</entry>
+              </row>
+
+              <row>
+                <entry>Job name:</entry>
+
+                <entry>The job name is typically a regular expression as well,
+                since the job name is generated dynamically. For example, a
+                compile job compile-54eB67e567e, could use "compile-" or
+                "compile-*" or the exact match "^compile-.$"</entry>
+              </row>
+
+              <row>
+                <entry>All:</entry>
+
+                <entry>Applies for all HPCC Systems components. The default
+                placements for pods delivered is [all]</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
+
+        <para>Regardless of the order the placements appear in the
+        configuration, they will be processed in the following order: "all",
+        "type", "target", and then "pod"/"job".</para>
+
+        <sect4>
+          <title>Mixed combinations</title>
+
+          <para>NodeSelector, taints and tolerations, and other values can all
+          be placed on the same pods: [list] both per zone and per node on
+          Azure <programlisting>placements:
+- pods: ["eclwatch","roxie-workunit","^compile-.*$","mydali"]
```

Comment:
but 'compile-.*$' is a regex, so is it that it does support regex's, but the example 'compile-*' is wrong?